<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Pie Chart </string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="2" >
    <widget class="QScrollBar" name="vSBar" >
     <property name="maximum" >
      <number>999</number>
     </property>
     <property name="singleStep" >
      <number>10</number>
     </property>
     <property name="pageStep" >
      <number>100</number>
     </property>
     <property name="value" >
      <number>500</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="chartFrame" >
      <property name="minimumSize" >
       <size>
        <width>300</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="zoomFactorLB" >
         <property name="text" >
          <string>Start position:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="startPositionSB" >
         <property name="maximum" >
          <double>360.0</double>
         </property>
         <property name="minimum" >
          <double>0.0</double>
         </property>
         <property name="singleStep" >
          <double>1.0</double>
         </property>
         <property name="value" >
          <double>0.0</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>170</width>
         <height>570</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <widget class="QScrollBar" name="hSBar" >
     <property name="maximum" >
      <number>999</number>
     </property>
     <property name="singleStep" >
      <number>10</number>
     </property>
     <property name="pageStep" >
      <number>100</number>
     </property>
     <property name="value" >
      <number>500</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="BarChart" />
 </resources>
 <connections/>
</ui>
